addons/base_import/models/base_import.py

```diff
@@ -1404,6 +1404,18 @@ def _import_image_by_url(self, url, session, field, line_number):
                 'error': e
             })
 
+    @api.model
+    def _stringify_date_like_objects(self, data, options={}):
```

Comment:
You shouldn't use a mutable object as argument. Here is ok because you never modify it but it is a bad practice (see https://docs.python-guide.org/writing/gotchas/).
Easier to force the existence of options.
```suggestion
    def _stringify_date_like_objects(self, data, options):
```